aiida/cmdline/params/options/commands/computer.py

```diff
@@ -109,6 +138,17 @@ def should_call_default_mpiprocs_per_machine(ctx):  # pylint: disable=invalid-na
     'Use 0 to specify no default value.',
 )
 
+DEFAULT_MEM_PER_MACHINE = OverridableOption(
+    '--mem-per-machine',
+    prompt='Default amount of memory per machine (MB).',
+    cls=InteractiveOption,
+    prompt_fn=should_call_default_mem_per_machine,
+    required_fn=False,
+    type=click.INT,
```

Comment:
SLURM doesn't support fractional numbers, we tested with @muhrin.

I think we will simply go for KB to avoid these kind of problems.

> is it possible that someone would need to specify a specific number of kilobytes and so has to specify a fractional number of MBs?

Well, this PR adds a default value. If one ever wants to add a specific number of kilobytes, they can do it at the job submission.

Reply:
> Well, this PR adds a default value. If one ever wants to add a specific number of kilobytes, they can do it at the job submission.

Sure, but the difference in typing restrictions on the CLI and on the API might be confusing. But anyway, with kB and then ints I am fine. Let's go with that